### PR TITLE
Add some helper docs for contributing to the project

### PR DESCRIPTION
**Pull request by swissspidy**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Versions**
+
+* Browser:
```
Comment (grappler) [inline .github/ISSUE_TEMPLATE.md]: Do you think the Browser version is needed?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+**Description**
+<!-- Please describe what you have changed or added -->
+
+**How has this been tested?**
+<!-- Please describe in detail how you tested your changes. -->
+<!-- Include details of your testing environment, tests ran to see how -->
+<!-- your change affects other areas of the code, etc. -->
+
+**Screenshots**
+If applicable, add screenshots that verify the behavior of your changes
```
Comment (grappler) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
If applicable, add screenshots to show the visual change.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+* Fork the `master` branch of the repository on GitHub
+* Run `composer install` to install necessary development tools requirements
+* Make changes to the forked repository
+    * Try to follow the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/), especially in terms of inline documentation.
+    * Verify that using `composer lint` and `composer format` to automatically fix most coding standards issues
+    * Ideally write unit tests for any code changes and verify them using `composer test`
+* Commit and push changes to your fork and [submit a pull request](https://github.com/wearerequired/traduttore/compare) to the `master` branch
+
+We try to review incoming pull requests as soon as possible and either merg them or make suggestions for some further improvements.
```
Comment (grappler) [inline CONTRIBUTING.md]:
```suggestion
We try to review incoming pull requests as soon as possible and either merge them or make suggestions for some further improvements. 
```

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (swissspidy) [reply .github/PULL_REQUEST_TEMPLATE.md]: FWIW I copied that sentence from the Gutenberg repo if you wanna suggest that change there as well :-)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (swissspidy) [reply .github/ISSUE_TEMPLATE.md]: Probably not right now but perhaps in the future when there are some UI components as well :-)

Comment (grappler) [reply .github/ISSUE_TEMPLATE.md]: We could always add it back in then 🤷‍♂️

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (grappler) [reply .github/PULL_REQUEST_TEMPLATE.md]: You seem to have used an older version. https://github.com/WordPress/gutenberg/tree/master/.github

**Pull request merged**